pkg/driver/controller.go

```diff
@@ -873,8 +873,18 @@ func (cs *controller) GetCapacity(
 	}
 
 	zfsNodesCache := cs.zfsNodeInformer.GetIndexer()
+
 	params := req.GetParameters()
+
 	poolParam := helpers.GetInsensitiveParameter(&params, "poolname")
+	poolParamPool, _ := func() (string, string) {
```

Comment:
Can we add a comment explaining why need to parse the pool name.

Reply:
Sure, no problem. I'll update the PR quick.